### PR TITLE
Follow naming conventions in id

### PR DESCRIPTION
**Pull request by olinux**
Renaming wild_type to wildType for the id to follow the naming conventions -> fix for #15

### CODE REVIEW

**Comment by lzehl** [comment]
@UlrikeS91 : I agree with Oli's suggestion (cf. also #15) Do you agree? if yes please feel free to merge.

**Comment by UlrikeS91** [comment]
@olinux and @lzehl, shouldn't the file name also be renamed to "wildType" then?

**Comment by lzehl** [comment]
@UlrikeS91 no, the entries for the "name" property stays at it is usually written. the ID and the type have conventions:
- ID: lowerCamelCase
- type: UpperCamelCase
Which is true for spaces, but only question here is what to do with dashes in names? Should they remain in the ID and type?
@olinux what do you think?

**Comment by olinux** [comment]
I think dashes should be generally avoided if anyhow possible. Otherwise, I would suggest to treat them just as other "filling" characters and therefore treat them as if they would be a space... meaning "foo-bar" becomes "fooBar"

**Comment by lzehl** [comment]
@olinux okay. we can follow this convention. For this PR : @UlrikeS91 is right, but I misunderstood her. Could you change the filename of this json-ld also to "wildType.jsonld" ?

File: terminologies/phenotype/wild_type.jsonld [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wild_type",
+  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wildType",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Phenotype",
   "name": "wild type",
```
Comment (lzehl) [inline terminologies/phenotype/wild_type.jsonld]: I've checked again. It's completely valid to use the notation "wildtype"
@olinux again, not your mistake but could you fix it ? and change the at_id as well?

File: terminologies/phenotype/wild_type.jsonld [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wild_type",
+  "@id": "https://openminds.ebrains.eu/controlledTerms/phenotype/wildType",
```
Comment (lzehl) [inline terminologies/phenotype/wild_type.jsonld]: see below (it's okay to use "wildtype" notation)

**Review by lzehl — CHANGES_REQUESTED** [review]

**Pull request merged**